main.py

```diff
@@ -9,9 +9,10 @@
     parser.add_argument('--folder', default="test_data/CarlaExs/")
     args = parser.parse_args()
     odp = OpenDriveParser()
-    onlyfiles = [f for f in listdir(args.folder) if isfile(join(args.folder, f))]
-    if ".DS_Store" in onlyfiles:
-        onlyfiles.remove(".DS_Store")
+    # onlyfiles = [f for f in listdir(args.folder) if isfile(join(args.folder, f))]
+    # if ".DS_Store" in onlyfiles:
+    #     onlyfiles.remove(".DS_Store")
     odp.parse_file(args.filename)
-    for f in onlyfiles:
-        odp.parse_file(args.folder+f)
+    print(odp.data.roads[1].lanes.laneSection[0])
+    # for f in onlyfiles:
+    #     odp.parse_file(args.folder+f)
```

Comment:
Uncomment these lines

Reply:
Is this just a Mac specific thing? Does it affect the actual running of the code at all? I don't think there is any harm in keeping it, but I think I commented this out because I thought it didn't really influence the functionality. Although I'm not sure why I didn't just delete it.